sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -5,6 +5,10 @@ extensions:
     - name: "openshift-ansible"
     - name: "aos-cd-jobs"
   actions:
+    - type: "forward_parameters"
+      parameters:
+        - ORIGIN_TARGET_BRANCH
```

Comment:
it wont, but at least until we fix this globally the PRs that point to release branches wont fail.